foxpuppet/windows/browser/notifications/addons.py

```diff
@@ -0,0 +1,90 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this file,
+# You can obtain one at http://mozilla.org/MPL/2.0/.
+
+from distutils.version import LooseVersion
+
+from foxpuppet.windows.browser.notifications import BaseNotification
+
+from selenium.webdriver.common.by import By
+
+
+class AddOnInstallBlocked(BaseNotification):
+    """Add-on install blocked notification."""
+
+    def allow(self):
+        """Allow the add-on to be installed"""
+        with self.selenium.context(self.selenium.CONTEXT_CHROME):
+            if self.window.firefox_version >= LooseVersion('53'):
+                # Firefox >= 53
+                self.root.find_anonymous_element_by_attribute(
+                    'anonid', 'button').click()
+            else:
+                self.root.find_anonymous_element_by_attribute(
+                    'anonid', 'button').find_anonymous_element_by_attribute(
+                        'anonid', 'button').click()
+
+
+class AddOnInstallConfirmation(BaseNotification):
+    """Add-on install confirmation notification."""
+
+    _addon_name_locator = (
+        By.CSS_SELECTOR, '#addon-install-confirmation-content label')
+    _cancel_locator = (By.ID, 'addon-install-confirmation-cancel')
+    _confirm_locator = (By.ID, 'addon-install-confirmation-accept')
+
+    @property
+    def addon_name(self):
+        """Provide access to the add-on name.
+
+        :returns: The add-on name.
+        """
+        with self.selenium.context(self.selenium.CONTEXT_CHROME):
+            label = self.root.find_element(*self._addon_name_locator)
+            return label.get_property('value')
+
+    def cancel(self):
+        """Cancel add-on install."""
+        with self.selenium.context(self.selenium.CONTEXT_CHROME):
+            if self.window.firefox_version >= LooseVersion('53'):
+                # Firefox >= 53
```

Comment:
Could these comments be a bit more descriptive? Just thinking from a docs point of view.